tree-sitter/src/TreeSitter/Language.hs

```diff
@@ -17,13 +17,12 @@ import           System.Directory
 import           System.FilePath.Posix
 import           TreeSitter.Symbol
 
-newtype Language = Language ()
-  deriving (Show, Eq)
+data Language
```

Comment:
I opted to represent `Language` thus to match what we do with other types that shouldn’t be copied (or have any Haskell representation aside from pointers to them).